docs/guide/introduction.md

```diff
@@ -97,4 +97,96 @@ The rest of the documentation will include many specific, practical examples of
 
 ## Content schemas
 
-Nulla vitae elit libero, a pharetra augue. Nullam quis risus eget urna mollis ornare vel eu leo. Maecenas sed diam eget risus varius blandit sit amet non magna. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Praesent commodo cursus magna, vel scelerisque nisl consectetur et.
+The content field schema is another key concept in Apostrophe. The "schema" simply refers to the content fields for a particular page, piece, or widget type. In addition to setting fields for the user interface, it supports data validation.
+
+Our blog post module's schema might look something like this:
+
+```javascript
+// modules/blog-post/index.js
+module.exports = {
+  extend: '@apostrophecms/piece-type',
+  fields: {
+    add: {
+      authorName: {
+        label: 'Author name',
+        type: 'string'
+      },
+      body: {
+        label: 'Blog post body',
+        type: 'area',
+        options: {
+          widgets: {
+            '@apostrophecms/rich-text': {}
+          }
+        }
+      }
+    },
+    group: {
+      blogFields: [ 'authorName', 'body' ]
+    }
+  }
+};
+```
+
+Content schemas are configured in the `fields` setting. In this case, `fields` has two subsections: `add`, where fields are added to the schema, and `group`, which organizes the fields for the user interface.
+
+Each property in the `add` object is a field you are including in the schema. Each property in `group` is a section of the interface, set to an array of fields to include in that section.
+
+See the reference documentation on [the `fields` setting](/reference/module-api/module-overview.md#fields) and [individual field types](/reference/field-types/) for more information.
+
+### Using existing field groups
+
+Fields that a piece type inherits will likely already be in field groups. This is certainly true for the default fields, `title`, `slug`, and `visibility`. You can add new fields into these groups and rearrange them if needed. There are a few things to keep in mind as you do.
+
+**It's fairly simple to see what the existing groups are.** Working with inherited fields and field groups is harder when you don't know what they are. You can log them in your terminal easily from the new module's [initialization function](/reference/module-api/module-overview.md#initialization-function).
```

Comment:
@abea I thought we wanted to avoid terms like "easily". Additionally, pointing out that its "hard" adds fluff between "its fairly simple" and "you can log them". If you want to keep the sentence about difficulty, it might be better do try this

```suggestion
Working with inherited fields and field groups can be difficult when you don't know what they are. Fortunately, it's fairly simple to identify existing groups by logging them in your terminal from the new module's [initialization function](/reference/module-api/module-overview.md#initialization-function).
```

Reply:
I'm using a structure of the leading bold sentences, but I'll change them to H4s to be clearer.

Good point about "easily." I hedged with "fairly simple," but we should remove the other.